src/cleanfigure.m

```diff
@@ -50,15 +50,15 @@ function cleanfigure(varargin)
 
   % Set up command line options.
   m2t.cmdOpts = m2tInputParser;
-  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'minimumPointsDistance', 1.0e-10, @isnumeric);
   m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'handle', gcf, @ishandle);
+  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'targetResolution', '15x9@600',@isstr);
```

Comment:
Because we (currently) only use the total number of points/pixels in the target image, ie. product of these 3 parameters, perhaps we should just use that as the parameter and rename it to something like `targetNumberOfPoints`? We can then have a note on calculating this parameter in the description .

Alternatively, we could expose the bare `tol` parameter and allow the user to do their own `figArea/targetNumberOfPoints` calculation. This gives the user more freedom but means a good default is not possible.

Maybe we could expose both:
  -  A `minimumFeatureArea` option for fine grained control to set manually the `tol` parameter of Visvalingam algorithm,
  -  A `targetResolution` option which selects a good `tol` for `minimumFeatureArea` based on the figure properties.

If we want to keep support for `minimumPointsDistance` with this function, having `tol=sqrt(2*minimumPointsDistance)` seems like a good choice, but of course won't be identical.